changelog/fragments/1686949146-update-status-command.yaml

```diff
@@ -0,0 +1,34 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: breaking-change
+
+# Change summary; a 80ish characters long description of the change.
+summary: status command, change human output to be a summary, added human_full output
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+# NOTE: This field will be rendered only for breaking-change and known-issue kinds at the moment.
+description: >-
+  status command has been changed, the default human output now uses a list format
+  and summaries output.  Full human output can be obtained with the new `human_full`
+
+# Affected component; a word indicating the component this changeset affects.
+component: agent
```

Comment:
[Question]
Should it be `agent` or `elastic-agent`?

Reply:
The docs don't actually do anything with this field today, so don't think too hard about it. In Beats this would just be the name of the Beat, for agent it can always be "Elastic Agent". We should probably template this into the fragment.